### DISCUSSION TITLE
Humidity in thermostat groups

### DISCUSSION THREAD

**Discussion by ciB89** [SukramJ/aiohomematic#1379] [Q&A] [answered]
I have a bunch of homematic ip etrv-b devices. I have bundled them in a group for each room and exposed them to HA. Interestingly the groups have the ability to carry information for humidity. I have several zigbee temperature sensors that measure humidity and I have passed the information on to HA and raspberrymatic. But unfortunately I do now see any possibility to write that value to the humidity attributes of that group. Is there a possibility to write that value the the thermostat group, either in raspberrymatic (via the addon or directly) or set that value in HA itself?

**Comment by Baxxy13** [comment]
Not possible in RaspberryMatic. 
You can't write external data to the goup. The datapoint is "read-only".

**Reply by ciB89** [reply]
Is there a virtual device that I could add to the group that I could set the hundity for?

I was able to add a virtual  wall thermostat via cuxD, but I cannot add it to the group.

**Reply by Baxxy13** [reply, marked as answer]
No, not on RaspberryMatic / CCU and the HA-Integration doesn't support CUxD.
Imho humidity is not necessary for a heating-groups because there is no benefit for humidity for the heating-function.

**Reply by ciB89** [reply]
That is technically correct, but it would be nice to have the information, like humidity, in one device (in HA or even when exposing to another assistant such as Google assistant) instead of having multiple devices. For example: 
![Screenshot_20240111-160402](https://github.com/danielperna84/hahomematic/assets/20285334/14e81353-ba89-4d65-bc3d-319b8d6e0b6a)
Here you can see that HA can offer information about "Luftfeuchtigkeit"/humidity on the thermostat card.

**Reply by Baxxy13** [reply]
I know. 
And it works if you have a HM/HmIP Wall-Thermostat in your group.
Maybe you could use some custom-card in HA which can show data of different devices.
But i don't know if this is possible. 

Warum quäle ich mich eigentlich in englisch ab wenn hier ein "Kinderzimmer" im Spiel ist... ?

**Reply by ciB89** [reply]
Ich dachte, weil man allgemein auf Englisch schreiben sollte. Aber egal.
Ich dachte ich kann ein virtuelles Wandthermostat (cuxD) der Gruppe hinzufügen, das geht aber nicht. Oder bin ich nur zu blöd dafür?

**Reply by Baxxy13** [reply]
Wie geschrieben, auf CCUx / RaspberryMatic kannst du dahingehend nichts machen. 
Auch nicht mit CUxD. 
Im HA hingegen ist ja fast alles möglich, man muss nur die Lösung finden oder selbst erarbeiten.
Da kann ich aber nicht helfen.
Möglicherweise hat jemand anderes eine Idee.
